docs/integration-guides/index.md

```diff
@@ -0,0 +1,118 @@
+
+### Account, Key, Seed and Wallet IDs
+
+Knowing how private and public keys are related to seeds and accounts is critical to building a safe and secure integration. With the internal node wallet, there is also a unique wallet ID that adds further security and must be considered if using that wallet for development/testing (see the wallets section below for details). Review the [Account, Key, Seed and Wallet IDs](the-basics.md#account-key-seed-and-wallet-ids) section for some additional details.
+
+If you need to generate or manage seeds or mnemonic phrases, see the [seeds](key-management.md#seeds) section for some key information about options as they may differ from other existing projects and standards.
+
+### Units
+
+There are two main units used: nano and raw. The ratio is:
+$$
+1 nano = 10^{30} raw
+$$
+
+All [QR code](the-basics.md#uri-and-qr-code-standards) setups and RPC calls use raw for the amounts, while any human interaction with units is done at the Nano level.
+
+### Other concepts
+
+The above concepts capture a minimum understanding to begin your integration journey, and although additional resources will be called out further in the various guides, don't be afraid to check out other resources throughout the documentation here.
+
+---
+
+## Accessing RPC calls
+
+Most integrations send and receive funds by making RPC calls to a Nano node. This requires either access to a public API or running your own node on one of the available networks. Depending on your goals, both are valid approaches with many options and levels of engagement.
+
+### Running your own node
+
+Running a node involves installing, configuring and maintaining software on a server, preferably on a stable cloud service for the best performance and uptime. This approach gives you more control at the cost of additional effort.
+
+If going this route, we encourage use of the existing test network for initial integrations. Head over to the [Running a node](../running-a-node/overview.md) guide and make your way through the overview and security pages before stepping through the node setup guide. Make sure you have "Test network" selected in all the example commands.
+
+!!! info "Production integration node should be non-voting"
+	When moving to production with a node on the main network, we recommend running dedicated, non-voting node for your integration. If you are interested in running a representative node to help further decentralize the network consensus, please setup a separate node for this purpose to ensure both operate as effectively as possible. See the [Voting as a Representative guide](../running-a-node/voting-as-a-representative.md) for further details.
+
+### Public APIs
+
+Access to public APIs for the main network is also available via third-party vendors and community members. These are only available for the main network and caution should be taken when evaluating SLAs and uptime in general.
```

Comment:
Maybe add that this is the preferred way of quickly testing something out or build a PoC?

Reply:
Added a note about this, thanks!